src/libostree/libostree-devel.sym

```diff
@@ -17,19 +17,10 @@
   Boston, MA 02111-1307, USA.
 ***/
 
-LIBOSTREE_2020.7 {
+LIBOSTREE_2020.8 {
 global:
-  /* Add symbols here, and uncomment the bits in
-   * Makefile-libostree.am to enable this too.
-   */
-  ostree_repo_static_delta_execute_offline_with_signature;
-  ostree_repo_static_delta_verify_signature;
-  ostree_bootconfig_parser_get_overlay_initrds;
-  ostree_bootconfig_parser_set_overlay_initrds;
-  ostree_sysroot_deploy_tree_with_options;
-  ostree_sysroot_stage_tree_with_options;
-  ostree_sysroot_stage_overlay_initrd;
-} LIBOSTREE_2020.4;
+  someostree_symbol_deleteme;
```

Comment:
Ahh right, heh. We don't actually want this one. :)

```
+ echo 'Checking that the example symbol wasn'\''t copy-pasted...'
+ test -f /home/jenkins/agent/workspace/ci_ostreedev_ostree_PR-2214-QHBFKSNOHAYNXHHEDBQV7R3ODIWRASPH7JEKS4KHXXHSHWDYL4CQ/src/libostree/libostree-devel.sym
+ assert_file_has_content_once /home/jenkins/agent/workspace/ci_ostreedev_ostree_PR-2214-QHBFKSNOHAYNXHHEDBQV7R3ODIWRASPH7JEKS4KHXXHSHWDYL4CQ/src/libostree/libostree-devel.sym someostree_symbol_deleteme
+ fpath=/home/jenkins/agent/workspace/ci_ostreedev_ostree_PR-2214-QHBFKSNOHAYNXHHEDBQV7R3ODIWRASPH7JEKS4KHXXHSHWDYL4CQ/src/libostree/libostree-devel.sym
+ shift
+ for re in "$@"
++ grep -e someostree_symbol_deleteme /home/jenkins/agent/workspace/ci_ostreedev_ostree_PR-2214-QHBFKSNOHAYNXHHEDBQV7R3ODIWRASPH7JEKS4KHXXHSHWDYL4CQ/src/libostree/libostree-devel.sym
++ wc -l
+ test 2 = 1
+ _fatal_print_file /home/jenkins/agent/workspace/ci_ostreedev_ostree_PR-2214-QHBFKSNOHAYNXHHEDBQV7R3ODIWRASPH7JEKS4KHXXHSHWDYL4CQ/src/libostree/libostree-devel.sym 'File '\''/home/jenkins/agent/workspace/ci_ostreedev_ostree_PR-2214-QHBFKSNOHAYNXHHEDBQV7R3ODIWRASPH7JEKS4KHXXHSHWDYL4CQ/src/libostree/libostree-devel.sym'\'' doesn'\''t match regexp '\''someostree_symbol_deleteme'\'' exactly once'
```

Reply:
I always trip over this part...basically the linker doesn't support an empty section and it also doesn't support a file entirely with comments.  I think in the past I commented out the include...